### PR TITLE
[AL-1921] Video Bounding Box NDJson Converter

### PR DESCRIPTION
**Pull request by jtsodapop**
Iteration of video bounding box that will allow the following:
`to_common`
`from_common`

This ticket does not include inclusion of nested classifications, which will be looked into in a different ticket (AL-1946)

### CODE REVIEW

File: labelbox/data/serialization/ndjson/objects.py [hunk 1]
```diff
@@ -211,6 +287,15 @@ def from_common(
...
     ) -> Union[NDLine, NDPoint, NDPolygon, NDRectangle, NDMask, NDTextEntity]:
         obj = cls.lookup_object(annotation)
+
+        #if it is video segments
+        if (obj == NDSegments):
+            return obj.from_common(
+                annotation,
+                data,
+                feature_schema_id=annotation[0][0].feature_schema_id,
+                extra=annotation[0][0].extra)
```
Comment (msokoloff1) [inline labelbox/data/serialization/ndjson/objects.py:297]: How does this work? annotation is cannot be indexed if it is actually an ObjectAnnotation.

Comment (jtsodapop) [reply labelbox/data/serialization/ndjson/objects.py:297]: Sorry, this is a List[List[VideoObjectAnnotation]]. I have updated the type hint to reflect this

File: labelbox/data/serialization/ndjson/objects.py [hunk 2]
```diff
@@ -221,14 +308,17 @@ def from_common(
 
     @staticmethod
     def lookup_object(annotation: ObjectAnnotation) -> "NDObjectType":
```
Comment (msokoloff1) [inline labelbox/data/serialization/ndjson/objects.py:310]: This type hint is also wrong

Comment (jtsodapop) [reply labelbox/data/serialization/ndjson/objects.py]: Thanks, also just updated

**Review by msokoloff1 — APPROVED** [review]

**Comment by hydak** [comment]
merging this with an override due to a Snyk issue

**Pull request merged**